src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const STATUS = 'bookstore/categories/statusChecked';
+
+const initialState = {
+  categories: [],
+};
```

Comment:
- [ Required ] Please consider setting the `initialState` to an array as specified in the project requirements, currently, the `initialState` is an object. You can do this by refactoring this code as shown below.
```JavaScript
 const initialState = []
```
## Project requirements
![Screenshot from 2023-02-03 00-23-06](https://user-images.githubusercontent.com/16392046/216452874-5f4d9a9a-094d-480c-91c2-276895673f20.png)